### ISSUE TITLE
Vue @click handler triggers false positive for click-events-have-key-events in axe-linter

### ISSUE THREAD

**Issue by JamesCatt** [dequelabs/axe-core#2943]
Axe-linter in VS code incorrectly reports the following Vue `@click` handlers as a failure of _click-events-have-key-events_:

```vue
<a href="#" @click="myHandlerFunc">My link text</a>
<button @click="myHandlerFunc">My button text</button>
```
Both of these will be triggered by keypress and do not require an explicit keypress event handler.

Product: axe Linter

Expectation: Not throw a false positive on Vue `@click` handler

Actual: Reports `@click` on a link as a failure of _click-events-have-key-events_

Motivation: This is a false positive

<pre><code>
axe-linter version: 4.1.1

Browser and Assistive Technology versions

For Tooling issues:
- Node version: 10.16.0
- Platform:  Mac
</code></pre>

**Comment by straker** [comment]
Thanks for the issue. We'll look into it

**Comment by WilcoFiers** [comment]
I think we're going to need to turn this rule off. Not sure what I was thinking there.

**Comment by JamesCatt** [comment]
> I think we're going to need to turn this rule off. Not sure what I was thinking there.

@WilcoFiers I had assumed it was meant to apply to scenarios like:

```vue
<div @click="myHandlerFunc">
```
No?
